src/Main.java

```diff
@@ -0,0 +1,137 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Main obj = new Main();
+
+        System.out.println("Метод findSymbolOccurance");
+        System.out.println("Символ зустрічається - " + obj.findSymbolOccurance("Java", 'a') + " раз/разів");
+
+        System.out.println();
+
+        System.out.println("Метод findWordPosition");
+        System.out.println("Result : " + obj.findWordPosition("Apollo", "pollo" ));
+        System.out.println("Result : " + obj.findWordPosition("Apple", "Plant" ));
+
+        System.out.println();
+
+        System.out.println("Метод stringReverse");
+        System.out.println("Result : " + obj.stringReverse("Hello"));
+
+        System.out.println();
+
+        System.out.println("Метод isPalindrome");
+        System.out.println(obj.isPalindrome("ERE"));
+        System.out.println(obj.isPalindrome("Allo"));
+
+        System.out.println();
+        System.out.println();
+
+        System.out.println("Гра - вгадай слово");
+        obj.guessWord();
+    }
+
+    public int findSymbolOccurance(String str, char ch) {
+
+        char[] letters = str.toCharArray();
+        int count = 0;
+
+        for ( int i = 0; i < letters.length; i++ ) {
+            if ( letters[i] == ch ) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public int findWordPosition(String source, String target) {
+
+        int index = source.indexOf(target);
```

Comment:
The logic of this method can be implemented in 1 line by using `indexOf()` method. It does all the necessary things for you including the case when -1 should be returned because of the absence of the target string.